### PR TITLE
Convert onGrab from passing a string to passing an object with series…

### PR DESCRIPTION
**Pull request by halkeye**
… and episode information

Refer to #759 for more details

### CODE REVIEW

File: src/NzbDrone.Core/Notifications/NotificationService.cs [hunk 1]
```diff
@@ -90,14 +90,18 @@ private bool ShouldHandleSeries(ProviderDefinition definition, Series series)
 
         public void Handle(EpisodeGrabbedEvent message)
         {
-            var messageBody = GetMessage(message.Episode.Series, message.Episode.Episodes, message.Episode.ParsedEpisodeInfo.Quality);
+            var grabMessage = new GrabMessage();
```
Comment (markus101) [inline src/NzbDrone.Core/Notifications/NotificationService.cs]: Use an object initializer instead:

```
var grabMessage = new GrabMessage
{
   Message = GetMessage(message.Episode.Series, message.Episode.Episodes, message.Episode.ParsedEpisodeInfo.Quality),
...
}
```

Like so: https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/History/HistoryService.cs#L132-L141

Comment (halkeye) [inline src/NzbDrone.Core/Notifications/NotificationService.cs]: fixed

Comment (halkeye) [inline src/NzbDrone.Core/Notifications/NotificationService.cs]: I was using the ondownload above as a template, but yea, object inititalizers++

**Comment by kayone** [comment]
can you squash this so we can merge it in,

Thanks,

**Comment by halkeye** [comment]
@kayone Squashed. I'll start working on the fixes for the other branch now

**Comment by kayone** [comment]
Thanks!
merged into develop!

**Pull request closed without merging**